clang/lib/Driver/SanitizerArgs.cpp

```diff
@@ -1463,17 +1504,18 @@ void SanitizerArgs::addArgs(const ToolChain &TC, const llvm::opt::ArgList &Args,
 }
 
 SanitizerMask parseArgValues(const Driver &D, const llvm::opt::Arg *A,
```

Comment:
Instead of extending this function, it would be cleaner to create another one.

Reply:
Done - I reverted the changes to `parseArgValues`, and created `parseArgCutoffs` and `parseArgValuesOrCutoffs`. (`parseArgValuesOrCutoffs` is needed because `parseSanitizeArgs` doesn't know whether an argument has cutoffs.)